src/svg/Exit_20.svg

```diff
@@ -1,9 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 56.3 (81716) - https://sketch.com -->
-    <title>Icons/Exit/20</title>
-    <desc>Created with Sketch.</desc>
-    <g id="Icons/Exit/20" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-        <path d="M10,9.29289322 L16.6464466,2.64644661 L17.3535534,3.35355339 L10.7071068,10 L17.3535534,16.6464466 L16.6464466,17.3535534 L10,10.7071068 L3.35355339,17.3535534 L2.64644661,16.6464466 L9.29289322,10 L2.64644661,3.35355339 L3.35355339,2.64644661 L10,9.29289322 Z" id="Combined-Shape" fill="#000000" fill-rule="nonzero"></path>
-    </g>
-</svg>
\ No newline at end of file
+<svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This doesn't appear to be centered within the frame.  Is that OK?

Reply:
trying to figure that out now